webpack/assets/javascripts/react_app/routes/Models/ModelsPage/components/ModelDeleteModal.js

```diff
@@ -16,11 +15,8 @@ const ModelDeleteModal = props => {
       enforceFocus
       submitProps={{
         url: `/api/v2/models/${id}`,
-        message: sprintf(
-          __('Hardware Model %s was successfully deleted'),
-          name
-        ),
-        onSucc: props.fetchAndPush,
+        message: sprintf(__('Hardware Model %s was successfuly deleted'), name),
+        onSuccess: props.fetchAndPush,
```

Comment:
Should add `submitBtnProps` here so the button text says "Delete" and the `bsStyle` is "danger"